rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -223,6 +234,35 @@ def scriptpubkey_to_btc_address(data: bytes) -> BTCAddress:
     return scriptpubkey_to_bech32_address(data)
 
 
+def query_blockstream_like_blockheight(
+        base_url: str,
+) -> BlockNumber:
+    """
+    Query blockheight from APIs similar to blockstream.info
+    Returns the blockheight
+    May raise:
+    - RemoteError if got problems with querying the API
+    """
+    url = f'{base_url}/blocks/tip/height'
+    response = retry_calls(
+        times=CachedSettings().get_query_retry_limit(),
+        location='',
+        handle_429=True,
+        backoff_in_seconds=4,
+        method_name=url,
```

Comment:
why not use something better for the method name? like `query_blockstream_like_height`.